packages/api/lib/cwSfExecutionEventUtils.js

```diff
@@ -3,14 +3,16 @@
 const get = require('lodash/get');
 const set = require('lodash/set');
 const StepFunctions = require('@cumulus/aws-client/StepFunctions');
-const log = require('@cumulus/common/log');
+const Logger = require('@cumulus/logger');
```

Comment:
non-actionable: Mild concern that this isn't in a typescripted module somewhere.    Really, though, we need to move the api lambdas out of API, and that's entirely out of scope for this PR.

Reply:
Yup, I looked to see if any of the stuff I wrote could be moved into any of the TS libs, but it made most sense here.